ratis-examples/src/main/java/org/apache/ratis/examples/filestore/FileStoreClient.java

```diff
@@ -141,6 +155,14 @@ public long write(String path, long offset, boolean close, ByteBuffer buffer)
     return WriteReplyProto.parseFrom(reply).getLength();
   }
 
+  public DataStreamOutput getStreamOutput(String path) {
+    final StreamWriteRequestHeaderProto header = StreamWriteRequestHeaderProto.newBuilder()
+        .setPath(ProtoUtils.toByteString(path))
+        .build();
+    RaftClientRequest request = client.createDataStreamHeaderRequest(Message.valueOf(header.toByteString()));
+    return client.getDataStreamApi().stream(request);
```

Comment:
FileStoreClient should not create RaftClientRequest by itself. It will be automatically created.  Just send the path.
```
    return client.getDataStreamApi().stream(ProtoUtils.toByteString(path).asReadOnlyByteBuffer());
```